cluster-provision/images/vm-image-builder/build-containerdisk.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_PATH=$(dirname "$(realpath "$0")")
+
+function cleanup() {
+  if [ $? -ne 0 ]; then
+    rm -f "$tar_file"
+    docker image rm -f "$full_image_tag"
```

Comment:
if image doesnt exists , it wont fail ?
if it can fail add || true please

Reply:
Since `-f` is mentioned `echo $?` returns `0` so we good